ani.ter-markosyan/repo/Lesson_3/exercise_09/exercise_09.py

```diff
@@ -0,0 +1,3 @@
+price = int(input("Enter the price of meal: "))
+tip = int (input("Enter the percent tip you want to leave: "))
+print(price * tip / 100, price + price * tip / 100)
```

Comment:
RE: User message should be the more proper way. Display what represents the printed values.